### ISSUE TITLE
Ship Fuel Calculations Incorrect

### ISSUE THREAD

**Issue by iProfessore** [MaFi-Games/Captain-of-Industry-issues#1248]
### Issue description

Ship fuel calculations are incorrect. With a full basic tank it has roughly 2000km distance available. I send that ship on a 400km journey. After that 400km journey there should be 1600km left in the tank but there isn't, what's left is only 1000km. the sums do not add up.

### Reproduction steps

send a ship on a journey with a full tank.
once it gets there compare the journey length with the fuel remaining

### Game version and build number

v0.5.4g

### Operating system

Windows

**Comment by MaFi-Games** [comment]
If there is exploration involved, it costs extra fuel as well.

**Comment by iProfessore** [comment]
Are you telling me that after a 400km trip there it costs 600km in fuel for exploration? that does not make any sense

**Comment by MaFi-Games** [comment]
Yes, exploration costs 550 km worth of fuel. It is mainly for balancing purposes.

**Comment by iProfessore** [comment]
Balancing what? It may make sense to you guys but it makes no sense at all to the player.
It takes longer to get somewhere than to explore it yet it costs more fuel to explore than to travel, it is backwards, not logical
